apps/fz_http/lib/fz_http/devices.ex

```diff
@@ -8,7 +8,7 @@ defmodule FzHttp.Devices do
   alias FzHttp.{Devices.Device, Repo, Sites, Telemetry, Users, Users.User}
 
   def list_devices do
-    Repo.all(Device)
+    Repo.all(from Device, preload: :user)
```

Comment:
actually I'm going to change it to preload in the live view tomorrow.

credo is affecting my judgement... I went too far to avoid some warnings.